ci/integration-tests.sh

```diff
@@ -1,24 +1,28 @@
+#!/usr/bin/env bash
 set -x
+
+if [[ -z "${INTEGRATION}" ]]; then
+  exit 0
+fi
+
 rm ~/.cargo/bin/cargo-clippy
 cargo install --force --path .
 
 echo "Running integration test for crate ${INTEGRATION}"
 
-git clone --depth=1 https://github.com/${INTEGRATION}.git checkout
-cd checkout
+git clone --depth=1 "https://github.com/${INTEGRATION}.git" checkout
+cd checkout || exit 1
 
-function check() {
 # run clippy on a project, try to be verbose and trigger as many warnings as possible for greater coverage
-  RUST_BACKTRACE=full cargo clippy --all-targets --all-features -- --cap-lints warn -W clippy::pedantic -W clippy::nursery  &> clippy_output
-  cat clippy_output
-  ! cat clippy_output | grep -q "internal compiler error\|query stack during panic\|E0463"
-  if [[ $? != 0 ]]; then
-    return 1
-  fi
-}
+RUST_BACKTRACE=full \
+cargo clippy \
+  --all-targets \
+  --all-features \
+  -- \
+  --cap-lints warn -W clippy::pedantic -W clippy::nursery \
+  > clippy_output 2>&1
 
-case ${INTEGRATION} in
-  *)
-    check
-    ;;
-esac
+cat clippy_output
+if grep -q "internal compiler error\|query stack during panic\|E0463" clippy_output; then
```

Comment:
Similar, do this transition correct?

Reply:
I think so. I tested it locally and it seemed to work.